docs/logagent/output-filter-geoip.md

```diff
@@ -0,0 +1,90 @@
+Title: Add Geo-IP information to logs 
+Description: Add Geo-IP information to logs. log enrichment 
+
+
+## Output filter: geoip
+
+This plugin adds Geo IP information to logs. 
+An everyday use case is to enrich web server logs (or any logs with IP addresses) with geographical information derived from those IP addresses. 
```

Comment:
An everyday use case is to enrich web server logs (or any logs with IP addresses) with geographical information derived from those IP addresses. 
=>
An everyday use case is to enrich web server logs, or any logs with IP addresses, with geographical information derived from those IP addresses. 

*Comment:*
Please try to never use brackets `()` when writing sentences. Much better to use commas `,` . The content flows much better and is easily readable.